tee-worker/scripts/litentry/worker-config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "rust_log": "info,integritee_service=debug,ws=warn,sp_io=error,substrate_api_client=warn,itc_parentchain_light_client=warn,jsonrpsee_ws_client=warn,jsonrpsee_ws_server=warn,enclave_runtime=debug,ita_stf=debug,its_rpc_handler=warn,itc_rpc_client=warn,its_consensus_common=debug,its_state=warn,its_consensus_aura=warn,aura*=warn,its_consensus_slots=warn,itp_attestation_handler=debug,http_req=debug,lc_mock_server=warn,itc_rest_client=debug,lc_credentials=debug,lc_identity_verification=debug,lc_stf_task_receiver=debug,lc_stf_task_sender=debug,lc_data_providers=debug,itp_top_pool=debug,itc_parentchain_indirect_calls_executor=debug,",
+  "execution_path": "/tmp/workers",
+  "workers": [
+    {
+      "valid": true,
+      "key_path": "/home/${USER}/tmp/workers/w0",
+      "flags": [
+        "--clean-reset",
+        "--ws-external",
+        "--mu-ra-external-address",
+        "localhost",
+        "--trusted-worker-port",
+        "2000",
+        "--untrusted-worker-port",
+        "3000",
+        "--mu-ra-port",
+        "3443",
+        "--untrusted-http-port",
+        "4545",
+        "--node-url",
+        "wss://rpc.rococo-parachain-sg.litentry.io",
+        "--node-port",
+        "443",
+        "--trusted-external-address",
+        "wss://localhost",
+        "--running-mode",
+        "mock",
+        "--enable-mock-server"
+      ],
+      "subcommand_flags": [
+        "--skip-ra",
+        "--dev"
```

Comment:
These looks like non-prod parameters

Reply:
In general, I think this config file should server as a template. It needs manual editing before running on target machine. How do you think so?